test/test.js

```diff
@@ -15,3 +17,27 @@ test('Basic test', (t) => {
     t.end()
   })
 })
+
+test('Cache directory is moved to new location', (t) => {
+  // Download to old cache directory location
```

Comment:
I don't know about "temporary". I think the migration code is going to stick around for at least six months after release, since not everyone immediately upgrades.